instrumentation/github.com/labstack/echo/otelecho/go.mod

```diff
@@ -9,13 +9,15 @@ require (
 	github.com/stretchr/testify v1.10.0
 	go.opentelemetry.io/contrib/propagators/b3 v1.36.0
 	go.opentelemetry.io/otel v1.36.0
+	go.opentelemetry.io/otel/sdk v1.36.0
```

Comment:
This still seems to add a dependency on the SDK?

Reply:
It is only a test dependency that is going to be pruned by modules depending on otelecho. See https://github.com/open-telemetry/opentelemetry-go-contrib/issues/7444 for more details.